app/src/main/java/com/framgia/soundclound/screen/detailgenre/ItemGenreDetailViewModel.java

```diff
@@ -0,0 +1,54 @@
+package com.framgia.soundclound.screen.detailgenre;
+
+import android.databinding.BaseObservable;
+import android.view.View;
+
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.util.Constant;
+
+/**
+ * Created by ADMIN on 1/6/2018.
+ */
+
+public class ItemGenreDetailViewModel extends BaseObservable {
+    private TrackClickListener mTrackClickListener;
+    private MoreInfoClickListener mMoreInfoClickListener;
+    private Track mTrack;
+
+    public ItemGenreDetailViewModel(Track track, TrackClickListener trackClickListener,
+                                    MoreInfoClickListener moreInfoClickListener) {
+        mTrack = track;
+        mMoreInfoClickListener = moreInfoClickListener;
+        mTrackClickListener = trackClickListener;
+    }
+
+    public String getTitle() {
+        return mTrack != null ? mTrack.getTitle() : "";
+    }
+
+    public String getArtist() {
+        if(mTrack != null)
+        return mTrack.getPublisherMetadata() != null ? mTrack.getPublisherMetadata().getArtist()
```

Comment:
ERROR: 'if' child have incorrect indentation level 8, expected level should be 12.